Collection.js

```diff
@@ -12,26 +12,29 @@ Invoices.byTimeRange = function(filter, sortBy, sortOrder,limit){
   }
   sortQuery[sortBy] = sortOrder;
 
-  let curDate = new Date();
+  let endDate = new Date();
   let startDate = new Date();
+  let daywrapper = moment(startDate);
   switch (filter) {
     default:
     case "today":
-      startDate.setDate(curDate.getDate()-1);
+     daywrapper.startOf('day'); 
       break;
     case "week":
-      startDate.setDate(curDate.getDate()-7);
+      daywrapper.startOf('week');
+      daywrapper.add(1, 'days');//begin on mondays
```

Comment:
might be better to move this into a configuration file for `moment`, the view should not need to know how moment.js is configured (which day is the start of the week)